src/pages/index.tsx

```diff
@@ -133,15 +133,15 @@ const cardsAndBillsProduct = [
       "/CB-1.avif",
       "/CB-2.avif",
     ],
-    showMoreImage: [],
+    backOfCardImage: [],
```

Comment:
seems like an optional field, I think we should represent it as such

Reply:
I to have all back side images inside of the data objects. But I will have it as an optional field.